module/VuFind/src/VuFind/Db/Service/FeedbackService.php

```diff
@@ -40,6 +43,36 @@
  */
 class FeedbackService extends AbstractService
 {
+    /**
+     * Db columnn name to Doctrine entity field mapper
+     *
+     * @var array
+     */
+    protected $map = [
```

Comment:
I'd give this property a more specific name, like fieldMap. It might also be worth reintroducing a support method that utilizes this code, like:

```
protected function mapField($name)
{
    return $this->fieldMap[$name] ?? $name;
}
```

This approach would allow you to remove all the rows from the map where the key and value are the same, since the default behavior would cover them. Then the map just becomes a list of special cases/exceptions, and the code might require less maintenance in future if new fields are added.